### DISCUSSION TITLE
Changing <title>

### DISCUSSION THREAD

**Discussion by mohammed-abdulaziz** [community/community#66714] [GitHub Education]
Deleted Github Classroom Repo

Question

### Body
After accepting my school assignment on github classroom, I deleted the repository by mistake from my Github page, and now when I click on the asssignment github link, I get error 404: Page not found message. How can I restore access to the repo? I tried going back to accept the assignment invitation, but it shows that I've already accepted it.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @mohammed-abdulaziz , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
